js/rxjs.md

```diff
@@ -0,0 +1,24 @@
+# Bloomfire RxJS Style Guide
+
+*Very much a work in progress*
+
+## Table of Contents
+
+  1. [Observable Naming](#observables)
+
+## observables 
+
+  - Files containing observables should be named in camelCase.
+  - Observable classes should be capitalized and MixedCase.
+  - Observable classes should substitute a trailing $ for the word "observable."
+  - Observable instances should be camelCase.
+
+  ```jsx
+  import SomeAPICall$ from 'app/observables/someAPICallObservable';
+  ````
+
+  ```jsx
+  const someAPICall = app.getInstance(SomeAPICall$);
```

Comment:
fwiw, I agree that the instance should have a $. Makes the distinction between data and streams easier :trollface: . 👍 for @pretzelface adding standards. Also, for some reason I keep getting emails about this repo.